sensor/common/scan/scan.go

```diff
@@ -225,19 +307,15 @@ func (s *LocalScan) enrichImageWithMetadata(errorList *errorhelpers.ErrorList, r
 			errs = append(errs, err)
 			continue
 		}
-		metadata.DataSource = &storage.DataSource{}
 
 		// Ensure the metadata is set on the image we pass to i.e. fetching signatures. If no V2 digest is available for the
 		// image, the signature will not be attempted to be fetched.
 		// We don't need to do anything on central side, as there the image will correctly have the metadata assigned.
 		image.Metadata = metadata
-		log.Infof("Received metadata for image %q with id %q using reg %q", image.GetName().GetFullName(), image.GetId(), reg.Name())
-		log.Debugf("Metadata for image %q with id %q: %v", image.GetName().GetFullName(), image.GetId(), metadata)
 		return reg
 	}
 
 	errorList.AddErrors(errs...)
```

Comment:
is this still needed here? This may blow up the logs if there are a ton of registries and mirrors and they all fail

Reply:
Assuming referring to `errorLIst.AddErrors(errs...)`? If so that itself wont blow up the logs (if referring to creating many lines) but will yield a potentially long line containing each error.  I believe it's needed if want to know why each failed.  

CRI-O does the same thing (concatenates each error), when all fail the long error is returned. For example, I encountered this during testing how mirroring works, when describing a pod, saw this event:

```
Failed to pull image "example.com/nginx@sha256:a97a153152fcd6410bdf4fb64f5622ecf97a753f07dcc89dab14509d059736cf": rpc error: code = Unknown desc = (Mirrors also failed: [34.23.22.68/docker/nginx@sha256:a97a153152fcd6410bdf4fb64f5622ecf97a753f07dcc89dab14509d059736cf: pinging container registry 34.23.22.68: Get "https://34.23.22.68/v2/": dial tcp 34.23.22.68:443: i/o timeout]): example.com/nginx@sha256:a97a153152fcd6410bdf4fb64f5622ecf97a753f07dcc89dab14509d059736cf: pinging container registry example.com: StatusCode: 404, <!doctype html>...
```